Presentation.Web/Controllers/API/SystemRelationController.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Web.Http;
+using Core.ApplicationServices.SystemUsage;
+using Core.DomainModel.ItSystemUsage;
+using Core.DomainModel.Result;
+using Presentation.Web.Extensions;
+using Presentation.Web.Infrastructure.Attributes;
+using Presentation.Web.Models.SystemRelations;
+
+namespace Presentation.Web.Controllers.API
+{
+
+    [InternalApi]
+    [RoutePrefix("api/v1/systemrelations")]
+    public class SystemRelationController : BaseApiController
+    {
+        private readonly IItSystemUsageService _usageService;
+
+        public SystemRelationController(IItSystemUsageService usageService)
+        {
+            _usageService = usageService;
+        }
+
+        [HttpPost]
+        [Route("")]
+        public HttpResponseMessage PostRelation([FromBody] CreateSystemRelationDTO relation)
```

Comment:
Er det nødvendigt for os at have forskellige DTO'er til at repræsentere en relation? Kunne vi nøjes med bare SystemRelationDTO?

Reply:
Tjaeh... jeg ved ikke lige om jeg vil have NamedEntityDTO med i CREATE.. ej heller "id"

Reply:
undrede mig også over named tingen .. tænkte vi ikke havde navne for det meste (jvnf senere kommentar)